ToDoApp/ToDoApp/CreateTaskView.swift

```diff
@@ -52,7 +57,10 @@ struct CreateTaskView: View {
             .toolbar {
                 ToolbarItem(placement: .navigationBarLeading) {
                     Button("Cancel") {
-                        createTaskViewModel.onCancelButtonTapped()
+                        let defaultValue = createTaskViewModel.onCancelButtonTapped()
+                        titleInput = defaultValue.titleInput
```

Comment:
Is there a reason to reset title and deadline values? The create task sheet is destroyed when it is dismissed and you will not get old values anyway